src/main/java/org/skriptlang/skript/bukkit/recipes/elements/SecRegisterRecipe.java

```diff
@@ -0,0 +1,272 @@
+		"\tset recipe ingredients to 3 diamonds, 3 emeralds and 3 iron ingots",
+		"\tset the recipe group to \"custom group\"",
+		"\tset the recipe crafting category to crafting category misc",
+		"\tset the recipe result item to diamond helmet named \"Heavenly Helm\"",
+	"",
+	"#Furnace, Campfire and Smoking follow same format as Blasting example",
+	"create new blasting recipe with the namespacekey \"my_recipe\":",
+		"\tset the recipe experience to 5",
+		"\tset the recipe cooking time to 10 seconds",
+		"\tset the recipe group to \"custom group\"",
+		"\tset the recipe cooking category to cooking misc",
+		"\tset the recipe input item to coal named \"Ash\"",
+		"\tset the recipe resulting item to gunpowder named \"Dust\"",
+	"",
+	"#Smithing Trim follows the same format, except for 'result item'",
+	"create smithing transform recipe with key \"my_recipe\":",
+		"\tset the recipe base item to diamond helmet",
+		"\tset the recipe template item to paper named \"Blueprint\"",
+		"\tset the recipe addition item to netherite ingot named \"Pure Netherite\"",
+		"\tset the recipe result to netherite helmet named \"Pure Helmet\"",
+	"",
+	"create a new stonecutting recipe with namespacekey \"my_recipe\":",
+		"\tset the recipe source item to cobblestone named \"Cracked Stone\"",
+		"\tset the recipe result to stone named \"Refurnished Stone\""
+})
+@Since("INSERT VERSION")
+public class SecRegisterRecipe extends Section {
+
+	private static final boolean RUNNING_1_20 = Skript.isRunningMinecraft(1, 20, 0);
+
+	static {
+		Skript.registerSection(SecRegisterRecipe.class, "(register|create) [a] [new] %*recipetype% with [the] (key|id) %string%");
+	}
+
+	private Expression<String> providedName;
+	private RecipeType providedType;
+	private Trigger trigger;
+	private Node thisNode;
+	private String thisScript;
+	public static Recipe lastRegistered;
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult, SectionNode sectionNode, List<TriggerItem> triggerItems) {
+		providedType = ((Literal<RecipeType>) exprs[0]).getSingle();
+		if (providedType == RecipeType.COOKING || providedType == RecipeType.CRAFTING || providedType == RecipeType.COMPLEX) {
+			Skript.error("You can not register a '" + providedType + "' recipe type.");
+			return false;
+		} else if (providedType == RecipeType.SMITHING && RUNNING_1_20) {
+			Skript.error("You can not register a 'smithing' recipe type on MC version 1.20+");
+			return false;
+		}
+		providedName = (Expression<String>) exprs[1];
+		AtomicBoolean delayed = new AtomicBoolean(false);
+		Runnable afterLoading = () -> delayed.set(!getParser().getHasDelayBefore().isFalse());
+		trigger = loadCode(sectionNode, "register recipe", afterLoading, providedType.getEventClass());
+		if (delayed.get()) {
+			Skript.error("Delays cannot be used within a 'register recipe' section.");
+			return false;
```

Comment:
Why not? Maybe I wanna add user input for these, there's nothing really breaking and we love handholding right?
```applescript
on jump:
	set {_player} to player
	register a new furnace recipe with id "why&*($^#&$(*^#&$*#":
		set recipe result to bedrock
		set recipe cooking time to 1 hour
		set recipe experience to 100
		set recipe group to "hand_holders"

		set {-RecipeBuilder::%{_player}%::waiting} to true
		send "What should be the recipe input (item), you have 5 seconds to respond":
		set {_now} to 5 second later
		while {-RecipeBuilder::%{_player}%::waiting} is true:
			if now >= {_now}:
				send "You didn't define the input in time, speed up your typing" to {_player}
				delete {-RecipeBuilder::%{_player}%::*}
				stop
			wait a tick
		set recipe input to {-RecipeBuilder::%{_player}%::waiting}
	delete {-RecipeBuilder::%{_player}%::*}

on chat:
	{-RecipeBuilder::%player%::waiting} is true
	set {_input} to uncolored message parsed as item
	if {_input} is set:
		set {-RecipeBuilder::%player%::waiting} to {_input}
		stop
	send "'%uncolored message%' is an invalid item"

```